config/fxdata/rules.cfg

```diff
@@ -71,65 +81,81 @@ EasterEggSpeechInterval = 20000
 PreserveClassicBugs = 
 
 [computer]
-; Maximum health percentage when creatures are considered to be put to temple to cure of disease.
+; Computer AI will pick up diseased creatures with health at or below this percentage and place them in the Temple to be healed of disease.
 DiseaseHPTemplePercentage = 100
 
 [creatures]
+; Game turns between health recovery when creatures are sleeping in their Lair. [0-255]
 RecoveryFrequency = 10
+; Unused
```

Comment:
if these are unused, fully getting rid of them makes more sense then saying Unused